en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,12 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keyword to content or media. They describe the content and allow to manage it.
+In the keyword panel, you can create or destroy a keyword.
+
+Usage
+-----
+
+When you fill meta for content or media, already created keyword can be choice or you can create new on the keyword field.
+
+For now, they are only used to front office in the "content list block" and "media list by keyword" to filter displaying content from a list.
```

Comment:
"for now" and "only" should not be present.
Simpler rephrase :
The content list block provides a way to filter the content by keywords.